src/scripts/ic8/tb/tb_analysis_2035.py

```diff
@@ -144,23 +144,93 @@ def get_tb_analysis(
+                'Actual_inc': pps.PARTNER['cases'] / pps.PARTNER["population"],
+                'GP_inc': pps.CF_forgraphs['incidence'],
+                'CF_inc': pps.CF_InfAve.portfolio_results['cases']['model_central'] /
+                          pps.CF_InfAve.portfolio_results['population']['model_central'],
+                'IC_inc': pps.IC.portfolio_results['cases']['model_central'] /
+                          pps.IC.portfolio_results['population']['model_central'],
+                'IC_LB_inc': pps.IC.portfolio_results['cases']['model_low'] /
+                             pps.IC.portfolio_results['population']['model_central'],
+                'IC_UB_inc': pps.IC.portfolio_results['cases']['model_high'] /
+                             pps.IC.portfolio_results['population']['model_central'],
+                'IC_LB_adj': (pps.IC.portfolio_results['cases']['model_low'])*0.9,
+                'IC_UB_adj': (pps.IC.portfolio_results['cases']['model_high'])*1.1,
+            }
+        )
 
-        # build whole df for export
-    whole_df = pd.concat(list_of_dfs, axis=0)
+    # Save df
+    tb_cases.to_csv('tb_cases_2035.csv')
+
+    tbh_deaths = pd.DataFrame(
+        index=pd.Index(list(range(2010, 2036)), name='Year'),
+        data={
+            'Actual': pps.PARTNER['deaths'],
+            'GP': pps.CF_forgraphs['deaths'],
+            'Counterfactual': pps.CF_InfAve.portfolio_results['deaths']['model_central'],
+            'IC': pps.IC.portfolio_results['deaths']['model_central'],
+            'IC_LB': pps.IC.portfolio_results['deaths']['model_low'],
+            'IC_UB': pps.IC.portfolio_results['deaths']['model_high'],
+            'pop_actual': pps.PARTNER['population'],
+            'pop_cf': pps.CF_InfAve.portfolio_results['population']['model_central'],
+            'pop_ic': pps.IC.portfolio_results['population']['model_central'],
+            'Actual_inc': pps.PARTNER['deaths'] / pps.PARTNER["population"],
+            'GP_inc': pps.CF_forgraphs['mortality'],
+            'CF_inc': pps.CF_InfAve.portfolio_results['deaths']['model_central'] /
+                      pps.CF_InfAve.portfolio_results['population']['model_central'],
+            'IC_inc': pps.IC.portfolio_results['deaths']['model_central'] /
+                      pps.IC.portfolio_results['population']['model_central'],
+            'IC_LB_inc': pps.IC.portfolio_results['deaths']['model_low'] /
+                         pps.IC.portfolio_results['population']['model_central'],
+            'IC_UB_inc': pps.IC.portfolio_results['deaths']['model_high'] /
+                         pps.IC.portfolio_results['population']['model_central'],
+            'IC_LB_adj': (pps.IC.portfolio_results['deaths']['model_low'])*0.9,
+            'IC_UB_adj': (pps.IC.portfolio_results['deaths']['model_high'])*1.1,
+        }
+    )
 
-    # save to csv
-    whole_df.to_csv(filename, index=False)
+    # Save df
+    tbh_deaths.to_csv('tbh_deaths_2035.csv')
```

Comment:
for saving files, I would put into `\sessions\...` to avoid it being tangled up with the scripts -- but this is very minor point!